templates/book/index.html.twig

```diff
@@ -128,11 +128,28 @@
                         </td>
                     </tr>
                 {% endif %}
+
+
+
+                <tr>
+                    <th>{{ 'Number of pages'|trans }}</th>
+                    <td>
+                        {{ component('InlineEditBook', {'book':book, 'field':'pageNumber'}) }}
+                    </td>
+                </tr>
+                <tr>
+                    <th>{{ 'UUID'|trans }}</th>
+                    <td>{{ book.uuid }}</td>
+                </tr>
```

Comment:
is it really necessary to display it everywhere? 99% of users won't care

Reply:
I added `{% if app.debug }` around it.